### DISCUSSION TITLE
Question about feature request # 3127 (inlay hints parameters name)

### DISCUSSION THREAD

**Discussion by A-kirami** [microsoft/pylance-release#4216] [General]
discuss #3127 
> This has been implemented.

I can't find a pylance setting like the one below.

[typescript parameter name inlay hints](https://code.visualstudio.com/docs/typescript/typescript-editing#_inlay-hints)
`typescript.inlayHints.parameterNames.enabled`

[rust-analyzer parameter name inlay hints](https://rust-analyzer.github.io/manual.html#configuration)
`rust-analyzer.inlayHints.parameterHints.enable`

**Comment by debonte** [comment]
We support inlay hints for three types of symbols which can be enabled/disabled separately:

- `python.analysis.inlayHints.functionReturnTypes` -- Enable/disable inlay hints for function return types.
- `python.analysis.inlayHints.variableTypes` -- Enable/disable inlay hints for variable types.
- `python.analysis.inlayHints.pytestParameters` -- Enable/disable inlay hints for pytest function parameters.
